frontends/ast/ast.cc

```diff
@@ -1428,8 +1430,9 @@ RTLIL::IdString AstModule::derive(RTLIL::Design *design, const dict<RTLIL::IdStr
 
 	std::string new_modname = modname;
 	if (has_interfaces)
+    {
 		new_modname += "$interfaces$" + interf_info;
-
+    }
```

Comment:
Same here.

Reply:
Actually it looks like these braces add no additional value?

Reply:
The braces are a leftover from the debugging process. You're completely right in saying that they're not necessary but I left them in to match the local code style.

Reply:
Generally, it is good practice to minimise the size of the PR so that it's easier to review.